store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/MutableStore.kt

```diff
@@ -0,0 +1,11 @@
+package org.mobilenativefoundation.store.store5
+
+import kotlinx.coroutines.flow.Flow
+
+interface MutableStore<Key : Any, CommonRepresentation : Any, NetworkWriteResponse : Any> : Store<Key, CommonRepresentation> {
```

Comment:
I think we should consider not encoding the network write response type into the store's signature